SourceS/sdl2_to_1_2_backports.h

```diff
@@ -1,13 +1,15 @@
 #pragma once

```

Comment:
Afaik the Xbox uses SDL2, so none of the changes to this file should be nessesery?

Reply:
As above, I'm currently using 1.2 but we do have both.

Reply:
ok, i'll revert some of the changes then. I'm thinking the Xbox port won't be working at this point since i just reverted everything about this file.